### ISSUE TITLE
Pagination on tables with Mongodb

### ISSUE THREAD

**Issue by melohagan** [Budibase/budibase#7355]
### Discussed in https://github.com/Budibase/budibase/discussions/7332

<div type='discussions-op-text'>

Pagination isn't possible in the MongoDB connector yet as we need to add support for **limit**.

Another consideration is thinking about how to handle pagination. 
Looking at this article <a href="https://www.codementor.io/@arpitbhayani/fast-and-efficient-pagination-in-mongodb-9095flbqr">here</a> using ObjectId and limit is probably the way to go over using skip.

Another good post here: https://medium.com/swlh/mongodb-pagination-fast-consistent-ece2a97070f3</div>

**Comment by aptkingston** [comment]
We would need to upgrade the mongo DB connector to be a "plus" datasource, since at the moment we just have a binary split of plus or non-plus. If that's even possible, it would be a large amount of work. Plus datasources at the moment support features like pagination, full search via lucene translation and both table and schema CRUD.

Eventually it would be better for datasources to support individual features (like just pagination for example) as it would enable us to improve individual datasources without requiring the full "plus" sepcification.

So basically for now, I don't think this will be possible. Of course there's always the possibility of hacking in some edge case, but I'm discounting that option.

**Comment by melohagan** [comment]
Closing in favour of: https://github.com/Budibase/budibase/issues/5387
